CONTRIBUTING.md

```diff
@@ -50,11 +50,11 @@ There are three categories of issues on the defaang GitHub issue tracker:
 
 If you want to help, make sure to _communicate your intent_, that is:
 
-1. If you want to tackle an issue, comment and say that you want to develop a PR. In the PR, please link the issue.
-2. If you want to tackle an issue, but along the way realize that you ran out of time and can't finish it: _communicate it_! This is an Open Source project, and retracting is no shame, but not communicating that you cannot finish something will hinder all other people involved.
+1. If you want to start working on an issue, comment and say that you want to develop a PR. In the PR, please link the issue.
+2. If you start working on an issue, but along the way realize that you ran out of time and can't finish it: _communicate it_! This is an Open Source project, and retracting is no shame, but not communicating that you cannot finish something will hinder all other people involved.
```

Comment:
plus, here, we're talking about the point at which people have already started - so that's the difference